ProblemsSolution.Tests/ProblemsSolutionTest.cs

```diff
@@ -0,0 +1,33 @@
+using NUnit.Framework;
+
+namespace ProblemsSolution.Tests
+{
+    [TestFixture]
+    public class Tests
+    {
+
+        [TestCase (new int[] {1,2,3,4,5}, ExpectedResult=6)]
+        [TestCase (new int[] {12, 22, 1, 3}, ExpectedResult=34)]
+        [TestCase (new int[] {3, 7, 0, 1}, ExpectedResult=0)]
+        public int SumArrayEvenNumbersTest(int[] numbers)
```

Comment:
Is it possible to add a short description of each test case? One must be able to see quickly what each test is about, so analyzing every input and extract the description from there is not always great.

Reply:
> Is it possible to add a short description of each test case? One must be able to see quickly what each test is about, so analyzing every input and extract the description from there is not always great.

Done 80f307ae3cfcdd0b3f05df8f44780a5638854953.